src/main/java/study/step3/Cars.java

```diff
@@ -15,10 +16,18 @@ public Cars(List<Car> cars) {
      * */
     public void move(){
         for(Car car : cars){
-            car.move(NumberChecker.checkCondition());
+            car.move(new RandomMoveStrategy());
         }
     }
     public List<Car> getCars(){
         return cars;
     }
+    public List<Car> getWinners(){
+        return selectMaxCar();
+    }
+    private List<Car> selectMaxCar(){
+        Optional<Car> maxCar = cars.stream().max((x,y)->Integer.compare(x.getPosition(),y.getPosition()));
+        return cars.stream().filter(x-> x.getPosition() == maxCar.get().getPosition())
+                .collect(Collectors.toList());
+    }
```

Comment:
```suggestion
        return cars.stream()
                .filter(x-> x.getPosition() == max)
                .collect(Collectors.toList());
    }
```
그러면 이 부분도 이렇게 표현할 수 있을 것 같습니다!